### DISCUSSION TITLE
Distribute backtest support now?

### DISCUSSION THREAD

**Discussion by OdinLin** [nautechsystems/nautilus_trader#1140] [Q&A] [answered]
I previously saw in the commit history that using dask to support distributed backtesting has been removed. May I ask what is the best way to do distributed backtesting now, or what is the suggestion to distributed backtesting.

**Comment by rterbush** [comment]
Very interested in this as well..

**Comment by OdinLin** [comment]
One way to achieve this is I think is use ray or dask, like previous issue comment https://github.com/nautechsystems/nautilus_trader/issues/354#issuecomment-865502748, but I wonder if there will be a distributed solution internal, which is mentioned in official website.

**Comment by limx0** [comment, marked as answer]
Hey @OdinLin and @rterbush - any of the previously mentioned libraries are suitable (dask, ray, etc).

This was removed from the core codebase due to:
- being quite trivial for users to do via these external libs (typically only a handful of line changes).
- not adding any value on top of these external libs
- ongoing maintenance requirements 

Instead, we're focusing efforts on ensuring interoperability so users can choose their preferred library.

**Reply by OdinLin** [reply]
Thanks, understand now.

**Comment by rterbush** [comment]
Thanks @limx0 very interesting to get that context...
